capstone/src/main/webapp/dashboard-loader.js

```diff
@@ -22,6 +22,16 @@ function getRooms() {
     });
 }
 
+function getSearchResults() {
+   fetch("/search?search=" + document.getElementById('search').value, {method: "GET", headers: new Headers({ID_TOKEN})}).then(response => response.json()).then((roomList) => {
+        const roomElement = document.getElementById("room-container");
+        roomElement.innerHTML = "";
+        for (room of roomList) {
+            roomElement.appendChild(createRoomDivElement(room));
+        };
+    }); 
```

Comment:
newline for each .then

Reply:
If you mean put each .then on a newline, I got it, but if you mean something else I'm not sure what you mean.